_data/projects/libreoffice-crash-reports.yml

```diff
@@ -0,0 +1,19 @@
+name: LibreOffice Crash Reports
+desc: Django-based website with statistics about LibreOffice crash reports
+site: http://crashreport.libreoffice.org/stats/
+tags:
+- libreoffice
+- office
+- desktop
+- crash
+- django
+- python
+- web
+- javascript
+- js
+- ui
+- qa
+- quality-assurance
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/mmohrhard/crash/issues
```

Comment:
I believe that should be done beforehand, because per the instructions in the README, this link should point to the list of such issues (and the label name above it should match as well). Or are *all* issues in that repo up-for-grabs? Note that http://up-for-grabs.net says: "This is a list of projects which have curated tasks **specifically for new contributors.**" (emphasis mine).